tests/test_mnist.py

```diff
@@ -93,10 +94,27 @@ def test_mnist_fedavg():
 
     num_clients = 2
     cfg = OmegaConf.structured(Config)
+    cfg.fed.args.num_local_epochs=2 
+
     model = CNN(1, 10, 28)
+    loss_fn = torch.nn.CrossEntropyLoss()
+
+    train_datasets, test_dataset = process_data(num_clients)
+
+    rs.run_serial(cfg, model, loss_fn, train_datasets, test_dataset, "test_mnist")
+
+def test_mnist_fedavg_lbfgs():
```

Comment:
I believe we do not have lbfgs available in main branch yet? Does this cause any error?

Reply:
It does not generate an error because I removed the lbfgs relative code, but I realize the function is just the same as test_mnist_fedavg() function. I will remove the function.